patches/chrome-browser-resources-side_panel-bookmarks-power_bookmarks_service.ts.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts
+index 1db2ac8548b77d181a23ea22f5ebff6eb8802e01..6b374163f3558b6e99e44c102afa794104e7af84 100644
+--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts
++++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts
+@@ -265,6 +265,7 @@ export class PowerBookmarksService {
+       bookmarks: chrome.bookmarks.BookmarkTreeNode[],
+       activeSortIndex: number): boolean {
+     let changedPosition = false;
++    if (activeSortIndex == 5) { return changedPosition }
```

Comment:
If you want, you can probably do this without a patch too, but as it's just one line I think it's probably fine. Up to you :smile:

Reply:
Hmm, I think subclassing this class and use it from `power_bookmarks_list.ts` would cause more patch lines.

Reply:
I was thinking you might just be able to monkey patch the function. Probably doesn't matter too much if it stays as a patch though.

```tsx
import PowerBookmarksService from '...'

const original = PowerBookmarksService.sortBookmarks
PowerBookmarksService.sortBookmarks = function(bookmarks, activeSortIndex) {
    if (activeSortIndex === 5) return false
    original.call(this, bookmarks, sortIndex)
}
```

also, just noticed but this should probably use `===` even if you keep it as a patch:

```suggestion
+    if (activeSortIndex === 5) { return changedPosition }
```

Reply:
Thanks updated! 👍🏼